inference/workflow/WorkflowContext.cs

```diff
@@ -30,6 +31,11 @@ public bool IsForEvaluation
         }
     }
 
+    public int GetAIChatEndpointIndex()
```

Comment:
I am not sure why you have Get... here and why not just make it a property?

Reply:
now a prop.